client/layout/guided-tours/config.js

```diff
@@ -108,14 +108,22 @@ function get( tour = 'main' ) {
 				linkUrl: 'https://learn.wordpress.com',
 			},
 		},
+		themes: {
+			version: '20160516',
+			init: {
+				description: 'Dummy tour for Themes',
+				text: i18n.translate( 'This is a dummy tour for Themes!' ),
```

Comment:
Let's remove the `translate` calls for any test tours, so translators don't needlessly translate :)